.prettierrc.yaml

```diff
@@ -0,0 +1,3 @@
+tabWidth: 2
+semi: true
```

Comment:
So I think this is the part that divides developers, those who like 4, 2 whatever spaces tab and those { at the end of the line and its own line hahaha

Reply:
LGTM.